src/test/java/de/adorsys/keycloak/config/service/ImportClientsIT.java

```diff
@@ -1971,6 +1996,342 @@ void shouldRemoveAuthzPoliciesForRealmManagement() throws IOException {
+        thrown = assertThrows(ImportProcessingException.class, () -> realmImportService.doImport(foundImport0));
+        assertThat(thrown.getMessage(), is("Cannot resolve 'unknowntype.resource.$unknown-id' in realm 'realmWithClientsForAuthzGrantedPolicies', the type 'unknowntype' is not supported by keycloak-config-cli."));
+
+        RealmImport foundImport1 = getFirstImport("49.1_update_realm_update_authz_policy_for_client_with_error_placeholder_realm-management.json");
+        thrown = assertThrows(ImportProcessingException.class, () -> realmImportService.doImport(foundImport1));
+        assertThat(thrown.getMessage(), is("Cannot find client 'missing-client' in realm 'realmWithClientsForAuthzGrantedPolicies' for 'client.resource.$missing-client'"));
+
+        RealmImport foundImport2 = getFirstImport("49.2_update_realm_update_authz_policy_for_idp_with_error_placeholder_realm-management.json");
+        thrown = assertThrows(ImportProcessingException.class, () -> realmImportService.doImport(foundImport2));
+        assertThat(thrown.getMessage(), is("Cannot find identity provider with alias 'missing-provider' in realm 'realmWithClientsForAuthzGrantedPolicies' for 'idp.resource.$missing-provider'"));
+
+        RealmImport foundImport3 = getFirstImport("49.3_update_realm_update_authz_policy_for_role_with_error_placeholder_realm-management.json");
+        thrown = assertThrows(ImportProcessingException.class, () -> realmImportService.doImport(foundImport3));
+        assertThat(thrown.getMessage(), is("Cannot find realm role 'Missing role' in realm 'realmWithClientsForAuthzGrantedPolicies' for 'role.resource.$Missing role'"));
+
+        RealmImport foundImport4 = getFirstImport("49.4_update_realm_update_authz_policy_for_group_with_error_placeholder_realm-management.json");
+        thrown = assertThrows(ImportProcessingException.class, () -> realmImportService.doImport(foundImport4));
+        assertThat(thrown.getMessage(), is("Cannot find group with path 'Missing group' in realm 'realmWithClientsForAuthzGrantedPolicies' for 'group.resource.$Missing group'"));
+    }
+
+    @Test
+    @Order(50)
+    void shouldNotTriggerErrorWhenReferencingInvalidUuidInFineGrainedAuthz() throws IOException {
+        // These scenarios do not use placeholders and instead reference objects by UUID - which do not need to exist.
+        // Keycloak accepts this and it sometimes even works (for objects that allow specifying UUID in creation and are created after the import)
+        // This is how to partially support fine-grained authz for types that are not supported yet by keycloak-config-cli
+        // keycloak-config-cli will log a warning, but otherwise the import succeeds
+
+        RealmImport foundImport0 = getFirstImport("50.0_update_realm_update_authz_policy_for_unknown_type_with_id_realm-management.json");
+        realmImportService.doImport(foundImport0);
```

Comment:
Let's have some asserts here. 

```suggestion
        assertDoesNotThrow(() -> realmImportService.doImport(foundImport0));
```